desktop/components/artist_page_cta/view.coffee

```diff
@@ -50,7 +52,13 @@ module.exports = class ArtistPageCTAView extends Backbone.View
     @$overlay.fadeIn 300
     @$banner.fadeOut 300
     fragment = qs.stringify @currentParams()
-    @afterAuthPath += "?#{fragment}" if fragment
+    # This handles the redirect for the new onboarding flow
+    @afterAuthPath += "?redirectTo=#{@artist.get('href')}"
+    @afterAuthPath += "#{fragment}" if fragment
```

Comment:
I think this needs an ampersand: "&#{fragment}"

Reply:
The `fragment` here is basically any works filter or tab you might be on, in an artist page, so something like: `?medium=works-on-paper`.

Maybe it should be a `"?#{fragment}"` then?

Reply:
I think `&` would be correct because I'm already adding the `?` for the `redirectTo` query param on line 56.

Reply:
Yea, but this isn't a separate query param (where of course you would use `&`), its actually _part_ of the `redirectTo` path.

Like `redirectTo=/artist/andy-warhol?medium=works-on-paper`. I don't think `redirectTo=/artist/andy-warhol&medium=works-on-paper` is correct.

Reply:
Be sure to also escape such characters, by URL encoding it, so that they don’t break the URL its embedded in.